build.sbt

```diff
@@ -7,7 +7,8 @@ skip in publish := true
 lazy val integrationTestSettings: Seq[Def.Setting[_]] = Defaults.itSettings ++ Seq(
   scalaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "scala",
   javaSource in IntegrationTest := baseDirectory.value / "src" / "test" / "java",
-  testOptions in Test := Seq(Tests.Argument(TestFrameworks.ScalaTest, "-l", "com.gu.test.tags.annotations.IntegrationTest"))
+  testOptions in Test := Seq(Tests.Argument(TestFrameworks.ScalaTest, "-l", "com.gu.test.tags.annotations.IntegrationTest")),
+  parallelExecution in IntegrationTest := false
```

Comment:
I did this because the it tests were having timeouts in futures, however I think giving sbt 2GB instead of 1.5GB sped things up.  I'll play with removing that again tomorrow.